hw2_1/hw2-1/Program.cs

```diff
@@ -0,0 +1,317 @@
+                            prefix = Console.ReadLine();
+                        }
+
+                        Console.WriteLine(
+                            $"There are {storage.HowManyStartsWithPrefix(prefix)} strings which start with this prefix");
+                        break;
+                    case "5":
+                        Console.Write("Enter string which you want check if it is in storage: ");
+                        string stringForCheckingIfItInStorage = Console.ReadLine();
+                        if (stringForCheckingIfItInStorage == null)
+                        {
+                            Console.Write("Wrong! There are no symbols! Enter string which you want to add: ");
+                            stringForCheckingIfItInStorage = Console.ReadLine();
+                        }
+
+                        bool isInStorage = storage.Contains(stringForCheckingIfItInStorage);
+                        if (isInStorage)
+                        {
+                            Console.WriteLine("There is entered string in storage!");
+                        }
+                        else
+                        {
+                            Console.WriteLine("There isn't entered string in storage!");
+                        }
+
+                        break;
+                    default:
+                        Console.WriteLine("You entered something wrong!");
+                        break;
+                }
+
+                Console.WriteLine("You can choose one of the options:\n" +
+                                  "1. Add string\n" +
+                                  "2. Remove string\n" +
+                                  "3. Find out how many strings are in storage\n" +
+                                  "4. Find out how many strings in storage starts with your entered string\n" +
+                                  "5. Find out if storage contains entered string\n" +
+                                  "6. Exit\n");
+                Console.Write("Write number 1, 2, 3, 4, 5 or 6: ");
+                choice = Console.ReadLine();
+            }
+        }
+    }
+
+    class BorStringStorage
+    {
+        class Vertex
+        {
+            public int howManyDescendants;
+            public bool isTerminal;
+            public Vertex[] arrayOfRelatedVertexes;
+
+            public Vertex(bool isTerminal, Vertex[] arrayOfRelatedVertexes)
+            {
+                howManyDescendants = 0;
+                this.isTerminal = isTerminal;
+                this.arrayOfRelatedVertexes = arrayOfRelatedVertexes;
+            }
+        }
+
+        private int size = 0;
+
+        public BorStringStorage()
+        {
+            Vertex[] currArrayOfVertexes = new Vertex[256];
+            _root = new Vertex(false, currArrayOfVertexes);
+        }
+
+        public int Size
+        {
+            get { return size; }
+            private set { size = value; }
+        }

```

Comment:
```suggestion
        public int Size { get; private set; } = 0;
```
Так короче, нагляднее, и не нужно явно заводить приватное поле `size`.